main/acle.md

```diff
@@ -465,6 +465,10 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
 
 * Added feature test macro for FEAT_SSVE_FEXPA.
 * Added feature test macro for FEAT_CSSC.
+* Added support for FEAT_FPRCVT intrinsics and `__ARM_FEATURE_FPRCVT`.
+* Added support for modal 8-bit floating point matrix multiply-accumulate widening intrinsics.
+* Added support for 16-bit floating point matrix multiply-accumulate widening intrinsics.
+* Added support for Brain 16-bit floating-point vector multiplication intrinsics.
```

Comment:
Maybe here you could add that this is in Alpha state. I dont see any where for sve_bfscake